api/src/main/resources/application.properties

```diff
@@ -32,9 +32,13 @@ logging.level.liquibase=WARN
 health.requiredSpareMemoryInMB=32
 health.urlsToCheck=http://www.google.com,http://www.facebook.com
 
+## -----------------------------------------------------------------------------------------------------  Event Logging
+eventlogger.kinesis.stream=bfd-insights-ab2d-dummy
+eventlogger.kinesis.region=US_EAST_1
```

Comment:
We probably want these be environment variables as usual, with default values as you specified here.